optax/contrib/_schedule_free_test.py

```diff
@@ -164,5 +164,16 @@ def run(opt):
     params_wrapper = run(opt_wrapper)
     chex.assert_trees_all_close(params_shortcut, params_wrapper)
 
+  @parameterized.parameters(*_OPTIMIZERS_UNDER_TEST)
+  def test_scalar_preservance(self, opt_name, opt_kwargs):
+    opt = getattr(alias, opt_name)(learning_rate=0.0, **opt_kwargs)
```

Comment:
perhaps call this base_opt ? otherwise both the base and the wrapper have the same name which is confusing